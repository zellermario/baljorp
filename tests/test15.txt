create SJ m1 0
create L m2 0
addNeighbor m1 1 m1
addNeighbor m2 2 m1

create Sarkkutato s m1

useskill s m2
pass s