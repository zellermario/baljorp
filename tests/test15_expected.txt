Fields:
SJ m1 (0) - object: x
	neighbours: [(1,m1)]
L m2 (0) - object: x
	neighbours: [(2,m1)]

Buildings:

Actors:
> Sarkkutato s (heat: 4) on m1
  Inventory: []